e2e-tests/test_prover_jobs.rs

```diff
@@ -0,0 +1,16 @@
+use e2e_tests::Orchestrator;
+use serde_json::json;
+
+extern crate e2e_tests;
+
+#[tokio::test]
+async fn test_sharp_job_completes() {
+    let mut orchestrator = Orchestrator::run("orchestrator", vec![]);
+    orchestrator.wait_till_started().await;
+
+    let client = reqwest::Client::new();
+
+    let create_job = json!({ "job_type": "ProofCreation", "internal_id": "12345" });
+
+    client.get(orchestrator.endpoint().join("/create_job").unwrap()).json(&create_job).send().await.unwrap();
```

Comment:
so this actually opens the same question we had yesterday if we should have endpoints only for testing because it seems like an anti pattern but I am not sure. i know this endpoint already existed before so it makes sense to use it. but actually, after the introduction of workers, this endpoint will be removed (my bad for not removing it). should the e2e tests be with workers?

Reply:
Yeah, I'll replace that with direct Mongo access (via Database trait)